components/PaywallModal.vue

```diff
@@ -331,7 +331,7 @@ const utmCampaign = computed(() => {
   return getRouteQuery('utm_campaign') || props.utmCampaign
 })
 const campaignId = computed(() => {
-  return getRouteQuery('utm_term') || utmCampaign.value
+  return getRouteQuery('utm_term') || getRouteQuery('utm_campaign')
```

Comment:
Since we have
https://github.com/likecoin/3ook-com/blob/8a16ac22e8311dd1ef989b10e1e147b45ff5af30/pages/pricing.vue#L145-L155

How about?
```suggestion
  return getRouteQuery('utm_term') || getRouteQuery('utm_campaign') || utmCampaign.value
```

Reply:
But `utmCampaign.value` would default to `pricing_page`, which is an invalid campaign id